.gitignore

```diff
@@ -121,6 +121,9 @@ aptos-move/aptos-vm-profiling/**/*.txt
 # Gas profiler outputs
 aptos-move/e2e-move-tests/gas-profiling
 
+# Atpos move generated files
+aptos-move/aptos-e2e-comparison-testing/test-data-mainnet-10m-15/rocks_txn_idx_db/*
```

Comment:
It may be better to ignore any `test-data-*` directories.

Reply:
I wasn't sure if this test-data are fixtures, and whether some test logic depends on them, so took the conservative approach. This brings up another thought I was having about our fork, having some CI to check the current tests in place. Let's keep them for now until we are sure.

Reply:
Yes, I think we should gut the workflows that are present upstream, selectively add testing for what we need and enable CI.